Top-Level Requirements.md

```diff
@@ -70,10 +72,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
     * 8.8.1 Define an “obstacle” representing the digging hole and send it to the navigation subsystem.
     * 8.8.2 Return the mining arm to the starting position.
+* 8.9 The mining subsystem **shall** expose a mechanism to reset arm postition.
```

Comment:
Typo in "postition". Also, we should specify exactly what kind of position it's resetting to: is it the position in the bin? Position where it doesn't interfere with the bin? Both? Something else?

Reply:
I kind of left that ambiguous because we didn't know then

Reply:
But I can clarify